mode/clike/clike.js

```diff
@@ -11,7 +11,30 @@
 })(function(CodeMirror) {
 "use strict";
 
-CodeMirror.defineMode("clike", function(config, parserConfig) {
+  function Context(indented, column, type, align, prev) {
```

Comment:
This file was retroactively wrapped in a module scope, hence the flat indentation. Please preserve it, so that indentation is at least coherent.